crates/turborepo-lib/Cargo.toml

```diff
@@ -9,19 +9,25 @@ license = "MPL-2.0"
 assert_cmd = "2.0.7"
 itertools = "0.10.5"
 pretty_assertions = "1.3.0"
+tempfile = "3.3.0"
 
 [dependencies]
 anyhow = { version = "1.0.65", features = ["backtrace"] }
+atty = "0.2"
 chrono = "0.4.23"
 clap = { version = "4.0.22", features = ["derive"] }
 clap_complete = "4.0.6"
+config = "0.13"
+dirs-next = "2.0.0"
 dunce = "1.0"
 env_logger = "0.10.0"
+lazy_static = "1.4.0"
 log = "0.4.17"
 predicates = "2.1.1"
 semver = "1.0"
 serde = { version = "1.0.145", features = ["derive"] }
 serde_json = "1.0.86"
 serde_yaml = "0.8.26"
+termcolor = "1.1.3"
```

Comment:
This is the third (to my knowledge) color library we use in this project, I think we should decide on a single one to use everywhere

https://crates.io/crates/console might be a good option
- has a function for inline styles https://docs.rs/console/0.15.4/console/fn.style.html
- you can enable/disable color globally https://docs.rs/console/0.15.4/console/fn.set_colors_enabled.html (for `FORCE_COLOR`)
  - also detects if colors are supported on its own

Reply:
I chose `termcolor` since we're already using `env_logger`'s color functionality that uses it: https://github.com/rust-cli/env_logger/blob/main/src/fmt/writer/termcolor/extern_impl.rs

Reply:
`console` looks good, but we would be adding a new color crate to our transitive deps. I'll look at switching out `colored` in favor of this.